doc/examples/denoise_patch2self.py

```diff
@@ -82,6 +90,14 @@
 Please set `shift_intensity=True` and `clip_negative_vals=False` by default to
 avoid negative values in the denoised output.
 
+The `b0_threshold` is used to separate the b0 volumes from the DWI volumes.
+Changing the value of the b0 threshold is needed is the b0 volumes in the
+`bval` file have a value greater than the default `b0_threshold`.
+
+The default value of `b0_threshold` in DIPY is set to 50. If using data
+such as HCP, the b0 volume tends to have a higher value, please set it to 100
+or more accordingly.
```

Comment:
Hmmm. The b-values that were used in the HCP 3T dataset are:  

```
array([   5.,  990.,  995., 1000., 1005., 1985., 1990., 1995., 2000.,
       2005., 2010., 2980., 2985., 2990., 2995., 3000., 3005., 3010.])
```

So, setting `b0_threshold=50` and setting `b0_threshold=100` would presumably give you exactly the same thing. I am also confused about the phrase `the b0 volume tends to have a higher value` -- it doesn't seem like this parameter would interact with the values of the image, only with the selection of the image volumes. Or am I missing something?

Reply:
Hi @arokem -- That is correct. If the bvals are less than 50 the `b0_threshold` at 50 and 100 would behave exactly the same way. Here are the bvals for HCP 7T for instance:
![image](https://user-images.githubusercontent.com/17598557/116840166-df0aa300-aba2-11eb-9f80-c1e03bc12087.png)

As you can see they are >=50 somewhere between 55 and 65. Although not common, I do want to warn users if they have such datasets. Let me try rephrasing the sentence to make it clearer.